frontend/src/styles/tokens.scss

```diff
@@ -25,6 +25,7 @@ $buttonFocusOutline:
 
 // Overwrite the Protocol colours with Nebula colours:
 // Purple:
+$color-purple-100: #5015cd;
```

Comment:
Just verifying: design is also using the term "Purple-100" for this colour? Because we didn't come up with these colours ourselves - they were aligned with design, so we'd have a consistent colour palette.

If not, then I'd either contact design and ask for a colour from the Nebula colour palette, or just pick the closed colour that already has a token available. (Or both :) )

Reply:
Thanks for the context, hard coded this color and removed it from tokens are talking with design. Left a comment saying its not in Nebula.